lib/osf-components/addon/components/resources-list/resource-card/template.hbs

```diff
@@ -2,7 +2,7 @@
     <p data-test-resource-card-type local-class='resourceType'>
         <img
             src={{this.resourceIcon}}
-            alt={{this.resourceTypeName}}
+            alt={{concat this.resourceTypeName ' ' (t 'osf-components.resources-list.badge')}}
```

Comment:
Does it make more sense for the translation string to take an argument than using the `{{concat}}` helper here?

Reply:
That's a good point! Some languages may put the "badge" descriptor before the resource type 🌍